sbi/inference/abc/mcabc.py

```diff
@@ -107,12 +110,14 @@ def __call__(
         assert (eps is not None) ^ (
             quantile is not None
         ), "Eps or quantile must be passed, but not both."
+        if kde_kwargs is None:
+            kde_kwargs = {}
 
         # Run SASS and change the simulator and x_o accordingly.
         if sass:
             num_pilot_simulations = int(sass_fraction * num_simulations)
             self.logger.info(
-                f"Running SASS with {num_pilot_simulations} pilot samples."
+                "Running SASS with %s pilot samples.", num_pilot_simulations
```

Comment:
Just curious, why are you making this change?

Reply:
Because of pylint :) there is a scenario where this end up in a security risk: https://bugs.python.org/issue46200